codespell_lib/_codespell.py

```diff
@@ -345,8 +345,9 @@ def parse_options(args):
                              'you\'d give "*.eps,*.txt" to this option.')
 
     parser.add_argument('-x', '--exclude-file', type=str, metavar='FILE',
-                        help='FILE with lines that should not be checked for '
-                             'errors or changed')
+                        help='ignore whole lines that match those '
+                             'in the file FILE. These lines in FILE '
```

Comment:
@kolyshkin 's improvement didn't make it into the main program help:
```suggestion
                             'in the file FILE. The lines in FILE '
```